docs/src/references/cli/tedge-http.md

```diff
@@ -0,0 +1,224 @@
+---
+title: "tedge http"
+tags: [Reference, CLI]
+sidebar_position: 6
+---
+
+# The tedge http command
+
+A `tedge` sub command to interact with the HTTP services hosted on the device by the Cumulocity mapper and the agent:
+
+- the [Cumulocity proxy](/references/cumulocity-proxy/)
+- the [file transfer service](/references/file-transfer-service)
+- the entity store.
+
+This command uses `tedge config` to get the appropriate host, port and credentials to reach these local HTTP services.
+So the same command can be used unchanged from the main device or a child device, with TLS or mTLS enabled or not.
+
+```sh title="tedge http"
+Send HTTP requests to local thin-edge HTTP servers
+
+Usage: tedge http [OPTIONS] <COMMAND>
+
+Commands:
+  post    POST content to thin-edge local HTTP servers
+  put     PUT content to thin-edge local HTTP servers
+  get     GET content from thin-edge local HTTP servers
+  delete  DELETE resource from thin-edge local HTTP servers
+  help    Print this message or the help of the given subcommand(s)
+
+Options:
+      --config-dir <CONFIG_DIR>  [env: TEDGE_CONFIG_DIR, default: /etc/tedge]
+      --debug                    Turn-on the DEBUG log level
+      --log-level <LOG_LEVEL>    Configures the logging level
+  -h, --help                     Print help (see more with '--help')
+```
+
+## Local HTTP services
+
+
+`tedge http` forwards requests to the appropriate local HTTP service using the URL prefixes.
+
+The requests are forwarded to the appropriate service depending on the URL prefix.
+
+- URIs prefixed by `/c8y/` are forwarded to the [Cumulocity Proxy](/references/cumulocity-proxy/)
+
+   ```sh title="Interacting with Cumulocity"
+   tedge http get /c8y/inventory/managedObjects
+   ```
+
+- URIs starting with `/tedge/file-transfer/` are directed to the [File Transfer Service](/references/file-transfer-service)
+
+   ```sh title="Transfering files across child devices"
+   tedge http put /tedge/file-transfer/target.txt --file source.txt
+   ```
+  
+- URIs starting with `/tedge/entity-store` are related to the Entity Registration
+
+   ```sh title="Listing child-devices and services"
+   tedge http get /tedge/entity-store/v1/entities
+   ```
+
+
+## Configuration
+
+For `tedge http` to be used from the main device or any child device, with TLS or mTLS enabled or not,
```

Comment:
```suggestion
For `tedge http` to be used from the main device or any client device, with TLS or mTLS enabled or not,
```
Used the word `client` instead of child, as this tool can be used from any client device that has `tedge` installed.

Reply:
Done https://github.com/thin-edge/thin-edge.io/pull/3357/commits/f71705349e51b269ab232631e3b7417334a342d8